components/ui/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,139 @@
+import { cva, VariantProps } from "class-variance-authority";
+import { Menu, Transition } from "@headlessui/react";
+import { Fragment } from "react";
+import { ChevronDownIcon } from "@heroicons/react/20/solid";
+
+const dropDownStyles = cva("", {
+  variants: {
+    items: {
+      primary: "bg-blue-200",
```

Comment:
Can we change this to use `bg-brand-600` and `bg-zinc-400`?
Like [here](https://github.com/ninjha01/storybook/blob/b2275d3dc3accc68af584f70c5650cf0a2f35222/components/ui/Button/Button.tsx#L9)?

One thing I didn't mention is that I'm using a [custom color in tailwind](https://github.com/ninjha01/storybook/blob/b2275d3dc3accc68af584f70c5650cf0a2f35222/tailwind.config.js#L17) so that we can quickly and easily change the styling per client.

Reply:
Also, can we add in dark mode selectors for primary and secondary? like `dark:bg-brand-foo and dark:bg-zinc-bar`?